apps/myaccount/src/configs/app.ts

```diff
@@ -163,6 +163,7 @@ export class Config {
                 window["AppUtils"]?.getConfig()?.ui?.isMultipleEmailsAndMobileNumbersEnabled,
             isPasswordInputValidationEnabled: window["AppUtils"]?.getConfig()?.ui?.isPasswordInputValidationEnabled,
             isProfileUsernameReadonly: window["AppUtils"]?.getConfig()?.ui?.isProfileUsernameReadonly,
+            passwordPolicyConfigs: window[ "AppUtils" ]?.getConfig()?.ui?.passwordPolicyConfigs,
```

Comment:
We don't use this value anywhere in myaccount, do we? if it's not used, better not to add it here.

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7185/commits/74c5553b10185d286b6af4950ea64b14563116d1